src/data/monitor.json

```diff
@@ -2,417 +2,417 @@
       "comment": "Placholder for an eventual CSS Block Layout Level 3 Module",
-      "lastreviewed": "2024-11-01"
+      "lastreviewed": "2025-01-01"
     },
     "https://drafts.csswg.org/css-contain-4/": {
       "comment": "No spec yet",
-      "lastreviewed": "2024-11-01"
+      "lastreviewed": "2025-01-01"
     },
     "https://drafts.csswg.org/css-floats-3/": {
       "comment": "No spec yet",
-      "lastreviewed": "2024-11-01"
+      "lastreviewed": "2025-01-01"
     },
     "https://drafts.csswg.org/css-page-template-1/": {
       "comment": "Marked as not ready for implementation",
-      "lastreviewed": "2024-11-01"
+      "lastreviewed": "2025-01-01"
     },
     "https://drafts.csswg.org/web-animations-css-integration/": {
       "comment": "Seems dormant",
-      "lastreviewed": "2024-11-01"
+      "lastreviewed": "2025-01-01"
     },
     "https://drafts.css-houdini.org/box-tree-api/": {
       "comment": "Marked as collection of ideas, no clear implementation plans yet",
-      "lastreviewed": "2024-11-01"
+      "lastreviewed": "2025-01-01"
     },
     "https://svgwg.org/specs/markers/": {
       "comment": "incomplete draft not being maintained, should be classified only as rough proposal per https://github.com/w3c/svgwg/issues/824#issuecomment-790946863",
-      "lastreviewed": "2024-11-01"
+      "lastreviewed": "2025-01-01"
     },
     "https://www.w3.org/TR/svg-markers/": {
       "comment": "incomplete draft not being maintained, should be classified only as rough proposal per https://github.com/w3c/svgwg/issues/824#issuecomment-790946863",
-      "lastreviewed": "2024-11-01"
+      "lastreviewed": "2025-01-01"
     },
     "https://svgwg.org/specs/paths/": {
       "comment": "incomplete draft not being maintained, should be classified only as rough proposal per https://github.com/w3c/svgwg/issues/824#issuecomment-790946863",
-      "lastreviewed": "2024-11-01"
+      "lastreviewed": "2025-01-01"
     },
     "https://www.w3.org/TR/svg-paths/": {
       "comment": "incomplete draft not being maintained, should be classified only as rough proposal per https://github.com/w3c/svgwg/issues/824#issuecomment-790946863",
-      "lastreviewed": "2024-11-01"
+      "lastreviewed": "2025-01-01"
     },
     "https://w3c.github.io/PFE/RangeRequest.html": {
       "comment": "being split-off from main spec https://github.com/w3c/PFE/pull/9/files; stub",
-      "lastreviewed": "2024-11-01"
+      "lastreviewed": "2025-01-01"
     },
     "https://dev.w3.org/html5/html-sourcing-inband-tracks/": {
-      "lastreviewed": "2024-11-01",
+      "lastreviewed": "2025-01-01",
       "comment": "no longer worked on and implementation status is unclear"
     },
     "https://wicg.github.io/document-isolation-policy/": {
       "comment": "Chrome status entry https://chromestatus.com/feature/5141940204208128 says no active development and no known standard positions yet",
-      "lastreviewed": "2024-11-01"
+      "lastreviewed": "2025-01-01"
```

Comment:
@tidoust Looking at this as part of an ongoing review of WICG documents, I realize we have a bug in the assumptions behind our monitoring update workflow: while the spec itself hasn't been updated, the implementation status has (it's now in origin trial)